source/module.h

```diff
@@ -58,8 +58,8 @@ napi_status aws_napi_dispatch_threadsafe_function(
     napi_value *argv);
 
 /**
- * Wrapper around napi_create_threadsafe_function that ensures it is a weak reference and cleans
- * up when the last node reference is cleared.
+ * Wrapper around napi_create_threadsafe_function,
+ * the theadsafe function needed to be released to clean up correctly
```

Comment:
I'm not sure what this change to the comment is trying to say, but I feel like this comment is less helpful now

Reply:
The previous comment is trying to make the threadsafe function a weak reference, and I removed the unref from the implementation. So, whenever we create a threadsafe function we need to release it.